.sync.yml

```diff
@@ -5,6 +5,8 @@
   docker_sets:
     - set: docker/centos-7
     - set: docker/ubuntu-14.04
+    - set: docker/ubuntu-16.04
```

Comment:
Can we get both Ubuntu16&18 removed from here and the travis file.

Reply:
done